fuzz.go

```diff
@@ -0,0 +1,11 @@
+package gonids
```

Comment:
can you add the copyright header here too please?

Reply:
done